src/commands/domain.js

```diff
@@ -181,7 +181,14 @@ export async function overview (params) {
 
   const applicationsWithDomains = await Promise.all(
     applications.map(async (app) => {
-      const domains = await getAllDomains({ id: app.ownerId, appId: app.id }).then(sendToApi);
+      const domains = await getAllDomains({ id: app.ownerId, appId: app.id })
+        .then(sendToApi)
+        .catch((error) => {
+          // if the user cannot access application domains, we simply act as if there were no domains
+          if (error?.response?.status === 403) {
+            return [];
+          }
+        });
```

Comment:
issue: I think we need to throw the error in the catch if we don't return. With the current code, the error is swallowed.